### PR TITLE
Fixed conversation to work with node v0.12

### PR DESCRIPTION
**Pull request by Prior99**

### CODE REVIEW

**Comment by botwillacceptanything** [comment]
#### :ballot_box_with_check: Voting procedure reminder:
To cast a vote, post a comment containing :+1:`:+1:`, or :-1:`:-1:`.
Remember, you **must :star:star this repo for your vote to count.**

All comments within this discussion are searched for votes, regardless of the time of posting.
You can cast as many votes as you want, but only the last one will be counted.
(You may consider editing your comment instead of adding a new one.)
Comments containing both :+1:up- and :-1:down-votes are disregarded.
Comments containing monkeys are disregarded. :monkey: :monkey_face: :see_no_evil: :hear_no_evil: :speak_no_evil:
PR authors automatically count as a :+1: vote.

A decision will be made after this PR has been open for **15** minutes (plus/minus **10** percent, to avoid people timing their votes), and at least **8** votes have been made.
A supermajority of **65%** is required for the vote to pass.

*NOTE: the PR will be closed if any new commits are added after:*
 46ffeb69346c3fa27ec954dd6846bf1bbc632df2

File: lib/integrations/irc.js [hunk 1]
```diff
@@ -23,6 +23,9 @@
                   destroy: function () {},
                 };
             }
+            else {
```
Comment (shalecraig) [inline lib/integrations/irc.js]: no need for this to be in an else statement

**Comment by shalecraig** [comment]
:-1:

**Comment by SpenserJ** [comment]
:+1: Even with the pointless else {}, I think its still worth an upvote.

**Comment by shalecraig** [comment]
:+1:

File: lib/integrations/irc.js (hunk 1 above)
Comment (Prior99) [inline lib/integrations/irc.js]: But it also doesn't hurt...

**Comment by PurHur** [comment]
:+1:

**Pull request closed without merging**